index.html

```diff
@@ -4,10 +4,47 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Personal Portfolio</title>
+    <title>Document</title>
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    
+    <section id="Home">
+        <nav>
+            <div class="logo">
+                <img src="" alt="">
+                <h2>My logo</h2>
+            </div>
+            <div class="navbar_links">
```

Comment:
- [ ] Kindly ensure that the list of icons is an `a` tag wrapped in a `li` tag then all wrapped in a `ul` tag.

something like this 

```
<ul>
  <li><a>home</a></li>
  <li><a>house</a></li>
  <li><a>yard</a></li>
  <li><a>apartment</a></li>
</ul>
```

This is good for semantic and user experience reasons respectively and also enables one to connect with the icons using the `href` attribute.